addons/better-img-uploads/userscript.js

```diff
@@ -0,0 +1,101 @@
+export default async function ({ addon, console }) {
+  let enabled = true; //Addon is enabled
+
+  addon.self.addEventListener("disabled", () => (enabled = false));
+  addon.self.addEventListener("reenabled", () => console.log((enabled = true)));
+
+  while (true) {
+    let ignore = false; //Have files already been changed?
+
+    let input = await addon.tab.waitForElement('input[type="file"][accept*=".svg"]', {
+      markAsSeen: true,
+    }); //Wait for the next image file input
+
+    input.addEventListener(
+      "change",
+      (e) => {
+        if (!enabled) return;
+        if (!ignore) (ignore = true) && onchange(e);
```

Comment:
No one-liner please. Also call stopPropagation here maybe?

Reply:
How is it now?